### PR TITLE
Connection quality

### PR DESCRIPTION
**Pull request by hiroshihorie**

### CODE REVIEW

File: lib/src/proto/livekit_rtc.pbjson.dart [hunk 1]
```diff
@@ -213,6 +213,15 @@ const SignalResponse$json = const {
       '9': 0,
       '10': 'roomUpdate'
     },
+    const {
```
Comment (boks1971) [inline lib/src/proto/livekit_rtc.pbjson.dart:216]: Wow, a bunch of magic numbers and strings. What are these?

**Review by boks1971 — APPROVED** [review]
@hiroshihorie looks fine to me, but there is a lot of stuff I do not understand in dart and there are some magic stuff in there. Needs another pair of :eyes: :-)

**Review by davidzhao — APPROVED** [review]
lgtm!

File: lib/src/proto/livekit_rtc.pbjson.dart (hunk 1 above)
Comment (davidzhao) [reply lib/src/proto/livekit_rtc.pbjson.dart:216]: these are generated protobuf files :)

**Comment by hiroshihorie** [comment]
Thanks Oniichans 🙏

**Pull request merged**